### PR TITLE
Pass jshint, better pasting, some small tweaks 

### PR DESCRIPTION
**Pull request by mikemaccana**
This looks like a bigger change than it is. Three parts:
- pass jshint - add some semis, put if bodies on separate lines, etc. Hope that's OK, I use jshint a lot and it's nice to keep quiet so that when it complains you know it's important
- when pasting, treat all individual values with the same steps - blank handling, duplicate handling, etc - that we were ding on individual inputs earlier
- when pasting, make the tags immediately - before we had to add a single trailing , to build the tags.

Hope that's OK. I've bumped the ver in both files to 0.8.1.

### CODE REVIEW

File: tags-input.js [hunk 1]
```diff
@@ -190,25 +216,30 @@
 				select(selectedTag.nextSibling);
 			}
 			else {
+				// If there are no tags yet, update the original input as the user types -
+				// so that users who only want one thing don't have to enter commas
+				if ( ! $('.tag', true).length ) {
+					originalInput.value = getValue();
+					originalInput.dispatchEvent(new Event('originalInput'));
```
Comment (developit) [inline tags-input.js]: Hmm, this won't work. `input` is an actual event in browsers, `originalInput` is not.

File: .jshintrc [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (developit) [inline .jshintrc:1]: Definitely not ready for `esnext` here, and we can just use `browser`:

``` json
{
    "browser" : true,
    "devel" : true,
    "boss" : true,
    "expr" : true
}
```

**Comment by mikemaccana** [comment]
Thanks @developit! Sorted the bad event, have adjusted PR according to comments.

**Comment by developit** [comment]
Manually merged ideas/chunks of this as part of #11. README update still needed though.

**Pull request closed without merging**